make/envoy-gateway.mk

```diff
@@ -0,0 +1,58 @@
+
+##@ Envoy Gateway
+
+## Targets to help install and configure EG
+
+EG_CONFIG_DIR = config/dependencies/envoy-gateway
+EG_NAMESPACE = envoy-gateway-system
+
+# egctl tool
+EGCTL=$(PROJECT_PATH)/bin/egctl
+EGCTL_VERSION ?= v1.0.0
+OS = $(shell uname -s | tr '[:upper:]' '[:lower:]')
+UNAME_P := $(shell uname -p)
+ifeq ($(UNAME_P),x86_64)
+	ARCH = amd64
+endif
+ifeq ($(UNAME_P),aarch64)
+	ARCH = arm64
+endif
+ifneq ($(filter armv5%,$(UNAME_P)),)
+    ARCH = armv5
+endif
+ifneq ($(filter armv6%,$(UNAME_P)),)
+    ARCH = armv6
+endif
+ifneq ($(filter armv7%,$(UNAME_P)),)
+    ARCH = arm
+endif
```

Comment:
Any reason not to use `uname -m` the same way it's done in the `get-egctl.sh`?

https://github.com/envoyproxy/gateway/blob/main/tools/hack/get-egctl.sh#L12-L25

On my system:
```
$ uname -p
unknown
```

Reply:
from `uname` manpage,
```
       -p, --processor
              print the processor type (non-portable)
       -s, --kernel-name
              print the kernel name
       -m, --machine
              print the machine hardware name
```

they all give `x86_64` in my machine.

using `uname (GNU coreutils) 9.4`

> I also just noticed that the ARCH var is already set in alerts.mk so it's currently getting set regardless.

good catch. I will refactor a bit.

Reply:
Can't say for sure what the difference between `-p` and `-m` is in my env.
```
$ uname --version
uname (GNU coreutils) 9.5

$ uname -p
unknown

$ uname -m
x86_64
```

Reply:
done, can you try getting egctl again?

Reply:
Nice yep all looks good now!